dpypelines/pipeline/dataset_ingress_v1.py

```diff
@@ -34,118 +35,121 @@ def dataset_ingress_v1(files_dir: str, pipeline_config: dict):
+            logger.error(f"Error while looking for supplementary distribution {supplementary_distribution}", 
+            err, 
+            data={"supplementary_distribution": supplementary_distribution,
+            "supplementary_distribution_patterns": supplementary_distribution_patterns}
+            )
             de_notifier.failure()
-            raise Exception(
-                message.unexpected_error(
-                    f"Error while looking for supplementary distribution {supplementary_distribution}",
-                    err,
-                )
-            ) from err
+            raise err
 
     # Get the positional arguments (the inputs) from the pipeline_config
     # dict and run the specified sanity checker for it
     args = []
     for match, sanity_checker in pipeline_config["transform_inputs"].items():
         try:
             input_file_path: Path = local_store.save_lone_file_matching(match)
+            logger.info("Successfully saved file that matches pattern", data={"input_file_path": input_file_path, "match": match})
         except Exception as err:
+            files_in_directory = local_store.get_file_names()
+            logger.error("Error occured while attempting to save matching pattern file.", 
+            err, 
+            data={"match": match, 
+            "pipeline_config": pipeline_config, 
+            "files_in_directory": files_in_directory})    
+
             de_notifier.failure()
-            printable_transform_details = json.dumps(
-                pipeline_config, indent=2, default=lambda x: str(x)
-            )
-            raise Exception(
-                message.pipeline_input_exception(
-                    printable_transform_details, local_store, err
-                )
-            ) from err
+            raise err
 
         try:
             sanity_checker(input_file_path)
+            logger.info("Successfully ran sanity check on input file path.", data={"input_file_path": input_file_path})
         except Exception as err:
+            files_in_directory = local_store.get_file_names()
+            logger.error("Error occured while running sanity checker on input file path.", 
+            err, 
+            data={"input_file_path": input_file_path, 
+            "pipeline_config": pipeline_config, 
+            "files_in_directory": files_in_directory})
+
             de_notifier.failure()
-            raise Exception(
-                message.pipeline_input_sanity_check_exception(
-                    pipeline_config, local_store, err
-                )
-            ) from err
+            raise err
 
         args.append(input_file_path)
 
     # Get the transform function and keyword arguments from the transform_details
     transform_function = pipeline_config["transform"]
     kwargs = pipeline_config["transform_kwargs"]
+    logger.info("Retrieved transform function and keyword arguments from transform details.", data={"pipeline_config": pipeline_config})
```

Comment:
can you include data= fields of something like "args_got" and "kwargs_got", so we can see the config as well as the args and kwargs taken from it.

doesnt matter now I know, but at some point we'll likely be getting them via something a little less explicit that just calling a key, always good to have the before and after where we can.